app/guid-file/template.hbs

```diff
@@ -102,9 +102,9 @@
                             <LoadingIndicator @dark={{true}} />
                         {{else}}
                             {{#if manager.inEditMode}}
-                                <div local-class='metadata-form-and-buttons'>
+                                <div local-class='metadata-form-and-buttons' data-test-edit-metadata-form>
                                     <div local-class='edit-metadata-form'>
-                                        <FormControls @changeset={{manager.changeset}} as |form|>
+                                        <FormControls data-test-edit-metadata-form @changeset={{manager.changeset}} as |form|>
```

Comment:
`data-test-edit-metadata-form` exists twice here. I think the DOM selector is only going to pick up the first one?

Reply:
Ok, this has been updated.